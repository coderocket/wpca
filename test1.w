x : int
	{x = X}
; skip
	{x = X}

