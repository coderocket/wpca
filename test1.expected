open misc
open util/integer

pred true { no none }
pred false { some none }

one sig State {
 x : Int
}

one sig Const {
 X : Int
}

pred obligation {
(State.x = Const.X) => {
State.x = Const.X
}
}

check { obligation }
